examples/todo/.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
aren't many of these already covered by the overrides in the parent `.eslintrc` above?

Reply:
yes i should clean it up. there is some odd merging tho.